docs/source/user-guide/sql/aggregate_functions.md

```diff
@@ -19,50 +19,293 @@
 
 # Aggregate Functions
 
-Aggregate functions operate on a set of values to compute a single result. Please refer to [PostgreSQL](https://www.postgresql.org/docs/current/functions-aggregate.html) for usage of standard SQL functions.
+Aggregate functions operate on a set of values to compute a single result.
 
 ## General
 
-- min
-- max
-- count
-- avg
-- sum
-- array_agg
+- [avg](#avg)
+- [count](#count)
+- [max](#max)
+- [mean](#mean)
+- [min](#min)
+- [sum](#sum)
+- [array_agg](#array_agg)
+
+### `avg`
+
+Returns the average of numeric values in the specified column.
+
+```
+avg(expression)
+```
+
+#### Arguments
+
+- **expression**: Column to operate on.
+
+#### Aliases
+
+- `mean`
+
+### `count`
+
+Returns the number of rows in the specified column.
+
+Count includes _null_ values in the total count.
+To exclude _null_ values from the total count, include `<column> IS NOT NULL`
+in the `WHERE` clause.
+
+```
+count(expression)
+```
+
+#### Arguments
+
+- **expression**: Column to operate on.
+
+### `max`
+
+Returns the maximum value in the specified column.
+
+```
+max(expression)
+```
+
+#### Arguments
+
+- **expression**: Column to operate on.
+
+### `mean`
+
+_Alias of [avg](#avg)._
+
+### `min`
+
+Returns the minimum value in the specified column.
+
+```
+min(expression)
+```
+
+#### Arguments
+
+- **expression**: Column to operate on.
+
+### `sum`
+
+Returns the sum of all values in the specified column.
+
+```
+sum(expression)
+```
+
+#### Arguments
+
+- **expression**: Column to operate on.
+
+### `array_agg`
+
+<!-- TODO: Add array_agg documentation -->
 
 ## Statistical
 
-- var / var_samp / var_pop
-- stddev / stddev_samp / stddev_pop
-- covar / covar_samp / covar_pop
-- corr
+- [corr](#corr)
+- [covar](#covar)
+- [covar_pop](#covar_pop)
+- [covar_samp](#covar_samp)
+- [stddev](#stddev)
+- [stddev_pop](#stddev_pop)
+- [stddev_samp](#stddev_samp)
+- [var](#var)
+- [var_pop](#var_pop)
+- [var_samp](#var_samp)
+
+### `corr`
+
+Returns the coefficient of correlation between two numeric values.
+
+```
+corr(expression1, expression2)
+```
+
+#### Arguments
+
+- **expression1**: First column or literal value to operate on.
```

Comment:
Likewise, I think all of these functions can take arbitrary expressions (literals, columns, or expressions like `col1 + col2`)